source/migration.txt

```diff
@@ -0,0 +1,176 @@
+.. _arch-center-migration:
+
+=========
+Migration
+=========
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: atlas architecture center
+   :description: Learn about live migrating data into Atlas from on-premises MongoDB databases using several automated processes.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: onecol
+
+You can migrate data from your on-premises MongoDB deployments to |service| using one of 
+a variety of methods. We recommend using :atlas:`|service| live migration </atlas/import/#migrate-or-import-data>` 
+when possible because it automates many of the tasks with the least amount of downtime, but you 
+can use other tools that accommodate the variety and complexity inherent to database migration. 
+
+Live Migration Overview
+----------------------- 
+
+|service| live migration automates moving data from on-premises MongoDB databases to |service|.
+You can pull data from an on-premises MongoDB database or push data using |com|, 
+but with either method, |service| live migration includes the following features:
+
+- The migration host always encrypts traffic to the |service| {+cluster+}. To encrypt 
```

Comment:
we can stop using the {+cluster+} source constant. you can make this change throughout the PR
```suggestion
- The migration host always encrypts traffic to the |service| cluster. To encrypt 
```